.gitignore

```diff
@@ -5,6 +5,11 @@
 npm
 npm-debug.log
 node_modules/
+package-lock.json
+bundle.js
+Photos/
+webpack.config.js
```

Comment:
We should commit webpack.config so all clones would not require a webpack config to be generated